miscellaneous/metaDataRemover/txCreator.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/blockchain"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/builders"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/core"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/data"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/interactors"
+	"github.com/ElrondNetwork/elrond-tools-go/miscellaneous/metaDataRemover/config"
+)
+
+type pkAddress struct {
+	privateKey []byte
+	address    core.AddressHandler
+}
+
+func getPrivateKeyAndAddress(pemFile string) (*pkAddress, error) {
```

Comment:
why not create a struct for this functionality in order to be able to test it

Reply:
Right, created interface for it and a struct